tests/test_solver.py

```diff
@@ -120,6 +120,41 @@ def _submit_and_check(self, solver, linear, quad, **kwargs):
 
         return results
 
+@unittest.skipUnless(config, "No live server configuration available.")
+class Utilities(_QueryTest):
+    """Test solver utility methods."""
+
+    def test_qpu_access_time_estimate(self):
```

Comment:
It would be great to run these tests on a mock solver, created from e.g. [`qpu_pegasus_solver_data`](https://github.com/dwavesystems/dwave-cloud-client/blob/442211d5279e9e6a96dc42175549caef66844970/dwave/cloud/testing/mocks.py#L183) -- where you can control estimation params. See [here](https://github.com/dwavesystems/dwave-cloud-client/blob/master/tests/test_solver_utils.py) for an example.

Reply:
If you feel like going down that route (using mock solver for testing the estimator), I would still leave a single/smoke test on a live solver (end-to-end test + SAPI test).

Reply:
Added mock for qpu_acess_time estimates